server.go

```diff
@@ -0,0 +1,79 @@
+package sqrl
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"sync"
+)
+
+var mutex = &sync.Mutex{}
+
+func com(w http.ResponseWriter, req *http.Request) {
+	mutex.Lock()
```

Comment:
So, after second though, we don't need these mutexes any where... Please get rid of all of them.